providers/celery/pyproject.toml

```diff
@@ -72,6 +72,9 @@ dependencies = [
 "cncf.kubernetes" = [
     "apache-airflow-providers-cncf-kubernetes>=7.4.0",
 ]
+"fab" = [
+    "apache-airflow-providers-fab"
+]
```

Comment:
@eladkal I'm not sure what introduced this change, but I think this is what is causing it to appear in the docs

Reply:
This was generated before my #49031  -> just rebasing and  regenerating all providers documentation should fix it.

Reply:
yep this is what I did